backend/Controllers/AdminController.cs

```diff
@@ -0,0 +1,26 @@
+using Microsoft.AspNetCore.Authorization;
+using Microsoft.AspNetCore.Mvc;
+using WhaleSpotting.Services;
+
+namespace WhaleSpotting.Controllers;
+
+[ApiController]
```

Comment:
Now that the Authorization PR (to check that users are logged in) has been merged in, I think an Authorize attribute should be added here to make sure the user has logged in, even before we check they are an admin

Reply:
I wonder if you can check they are logged in and an admin right up here at the top of the controller? then we won't have to do it separately for each endpoint

Reply:
I think one [Authorize(Roles="Admin")] here is enough as it implies a user needs authentication already with it